app/grants/models.py

```diff
@@ -1331,7 +1335,7 @@ def presave_contrib(sender, instance, **kwargs):
 
 def next_month():
     """Get the next month time."""
-    return localtime(timezone.now() + timedelta(days=30))
+    return (timezone.now() + timedelta(days=30))
```

Comment:
Why are we doing this ?

Reply:
In my env this line throws one exception due to localtime. I'll rollback this.